src/map/lua/luautils.cpp

```diff
@@ -311,7 +311,7 @@ namespace luautils
             }
         }
 
-        if (gLoadAllLua) // Load all lua files (for sanity testing, no need for during regular use)
+        if (true) // Load all lua files (for sanity testing, no need for during regular use)
```

Comment:
Debug testing left in

Reply:
Yup this is why I asked to look for stuff that doesn't belong. I was stuck on this branch for so long I had to do development in it too, i knew i'd miss something